vars/setupCommonPipelineEnvironment.groovy

```diff
@@ -31,15 +32,11 @@ import groovy.transform.Field
 @GenerateDocumentation
 void call(Map parameters = [:]) {
 
-    handlePipelineStepErrors (stepName: STEP_NAME, stepParameters: parameters) {
-
         def script = checkScript(this, parameters)
 
-        prepareDefaultValues script: script, customDefaults: parameters.customDefaults
-
-        String configFile = parameters.get('configFile')
+        prepareDefaultValues script: script, customDefaults: parameters.customDefaults, projectConfig: parameters.projectConfig
```

Comment:
Can we use `DefaultValueCache.prepare` directly and let `prepareDefaultValueCache` die?

Reply:
yes, of course. I wanted to stay backward compatible. There are more thing we can/should change. E.g. `cpe.configuration` can now be replaced easily by getting it from `DefaultValueCache`.

For use cases without any customer landscape (just or defaults and a project config) I guess there is no need for calling any initialisation step.